intelmq/lib/harmonization.py

```diff
@@ -436,9 +436,9 @@ def __parse(value: str) -> Optional[str]:
         return utils.decode(value)
 
     @staticmethod
-    def parse_utc_isoformat(value: str, return_datetime: bool = False) -> Union[datetime.datetime, str]:
+    def from_isoformat(value: str, return_datetime: bool = False) -> Union[datetime.datetime, str]:
```

Comment:
Do we consider these methods a publicly-facing interface of IntelMQ? If yes, considering semantic versioning guidelines, such changes require a new major version. If not, we can go forward with the rename.

Reply:
Thank you, good point. It is probably safer to consider it public interface. I do not wish to break someone else's code. I will add the old functions for compatibility purposes and mark them for deletion with next major release.